src/Controller/StatisticController.php

```diff
@@ -316,31 +316,31 @@ public function saturday (Request $request, Session $session)
         // 		--------------------------		Affichage du tableau de résultat		--------------------
         if ($tab) {
            
-            foreach ($heures_tab_global as &$v) {
+            foreach ($heures_tab_global as $v) {
```

Comment:
Attention, il ne s'agit pas ici de corriger une typo !

On change le fonctionnement des boucles en passant d'une itération sur référence à une itération sur variable.

Une page pour voir à quel point le fonctionnement est différent: 
https://stackoverflow.com/questions/3307409/php-pass-by-reference-in-foreach

En regardant le code legacy, je vois qu'on était bien à la base sur une itération sur variables, mais que ce comportement a été changé par le commit 7eb1950069ef3fd1a64d5020a853b237c4ece3e0 Symfonize saturdays stats page du Mon Mar 22 10:47:55 2021

On devrait a minima savoir pourquoi ce changement a été fait.
J'ai le sentiment que ce n'était pas voulu, et que ça a possiblement induit des effets de bord.
Je pense que remettre l'itération sur variable est la chose à faire, mais que des tests unitaires sur ce Controller seraient les bienvenus.